src/pages/LoginGoogle/LoginForm.jsx

```diff
@@ -0,0 +1,137 @@
+import { useState } from "react";
+// import { BtnLoginGoogle } from "./GoogleLogin";
+import { LoginGoogle } from "./LoginGoogle";
+// import { googleLogout, useGoogleLogin } from "@react-oauth/google";
+import axios from "axios";
+// import PropTypes from "prop-types";
+
+import { APP_ROUTES } from "../../utils/constants";
+import { useNavigate } from "react-router-dom";
+import css from "./LoginForm.module.css";
+import { API_ROUTES } from "../../utils/constants";
+// import { styled } from "styled-components";
+
+const LoginForm = () => {
+  const navigate = useNavigate();
+  // const [form, setForm] = useState({
+  //   email: "",
+  //   password: "",
+  // });
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  //
+  // const handleChange = (event) => {
+  //   setForm({
+  //     ...form,
+  //     [event.target.id]: event.target.value,
+  //   });
+  // };
+
+  // const handleSubmit = (event) => {
+  //   event.preventDefault();
+  //   // if [event.target.id] =
+  //
+  //   alert(form.email + " " + form.password + " " + `${event.currentTarget.lastChild }  ${event.target.id}` );
+  // };
+
+  const signUp = async () => {
+
+    event.preventDefault();
+    try {
+      // setIsLoading(true);
+      const response = await axios({
+        method: "POST",
+        url: API_ROUTES.SIGN_UP,
+        data: {
+          email,
+          password,
+        },
+      });
+      if (!response?.data) {
+        console.log("Something went wrong during signing up: ", response);
+        return;
+      }
+      navigate(APP_ROUTES.SIGN_IN);
+    } catch (err) {
+      console.log("Some error occured during signing up: ", err);
+    } finally {
+      // setIsLoading(false);
+    }
+  };
+
+  const signIn = async () => {
+
+    event.preventDefault();
+    try {
+      // setIsLoading(true);
+      const response = await axios({
+        method: "post",
+        url: API_ROUTES.SIGN_IN,
+        data: {
+          email,
+          password,
+        },
+      });
+      if (!response?.data?.token) {
+        console.log("Something went wrong during signing in: ", response);
+        return;
+      }
+      storeTokenInLocalStorage(response.data.token);
+      navigate(APP_ROUTES.HOME);
```

Comment:
to zapisywanie w storze nie jest nigdzie zdefiniowane i pewnie przez to logowanie nie jest w stanie przejść dalej